pkg/function/restore_rds_snapshot.go

```diff
@@ -237,7 +235,7 @@ func postgresRestoreCommand(pgHost, username, password string, dbList []string,
 		fmt.Sprintf(`
 		export PGHOST=%s
 		kando location pull --profile '%s' --path "%s" - | gunzip -c -f |%s psql -q -U "${PGUSER}" %s
-		`, pgHost, profile, fmt.Sprintf("%s/%s", backupArtifactPrefix, backupID), replaceCommand, dbList[0]),
+		`, pgHost, profile, fmt.Sprintf("%s/%s", backupArtifactPrefix, backupID), replaceCommand, postgres.DefaultConnectDatabase),
```

Comment:
```suggestion
		`, pgHost, profile, fmt.Sprintf("%s/%s", backupArtifactPrefix, backupID), replaceCommand, postgres.defaultConnectDatabase),
```

Reply:
@akankshakumari393 the const is declared in another package. To import and use the const from different package, it needs to be exported.